cinema/views.py

```diff
@@ -167,6 +185,24 @@
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type={"type": "string"},
+                description="Filter by date (ex. ?date=2024-31-10)",
```

Comment:
The date format in the description should be corrected to 'ex. ?date=2024-10-31' to match the standard YYYY-MM-DD format.